modules/cache.py

```diff
@@ -20,12 +21,21 @@ def get(
     if method is None:
         method = "POST" if json else "GET"
 
-    return requests.request(
-        method=method,
-        url=url,
-        headers=headers,
-        json=json,
-        files=files,
-        data=data,
-        timeout=2
-    )
+    try:
+        return requests.request(
+            method=method,
+            url=url,
+            headers=headers,
+            json=json,
+            files=files,
+            data=data,
+            timeout=2
+        )
+    except requests.exceptions.Timeout as exception_response:
+        api_error_exception = ApiErrorResponse("Timeout")
```

Comment:
Since a timeout is something that the API layer should explicitly expect, I think we should be more declarative here and create an `ApiTimeoutError` class.

Then we can simply do:

``` python
return ApiTimeoutError('The request to {} timed out'.format(url))
```

And we don't need to worry about hacking `code` and `status` onto another error type.